Song/VerseWriter.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song
+{
+    public class VerseWriter
+    {
+        private List<string> _lines;
+        private string _lastAnimalName;
+        public VerseWriter() 
+        {
+            _lines = new List<string>();
+            _lastAnimalName = string.Empty;
+        }
+        public string WriteVerse(string animalName, string animalUniqueLine)
+        {
+            if(_lines.Count == 0)
+            {
+                _lines.Add($"There was an old lady who swallowed a {animalName}.");
+                _lines.Add($"I don't know why she swallowed a {animalName} - perhaps she'll die!");
+                _lastAnimalName = animalName;
+                return ParseVerse();
+            }
+
+            _lines[0] = $"There was an old lady who swallowed a {animalName};";
+
+            if (_lines.Count == 2) 
+            {
+                _lines.Insert(1, animalUniqueLine);
+            }
+            else
+            {
+                _lines[1] = animalUniqueLine;
+            }
+
+            _lines.Insert(2, $"She swallowed the {animalName} to catch the {_lastAnimalName}");
+
+            _lastAnimalName = animalName;
+            return ParseVerse();
+        }
+
+        private string ParseVerse()
```

Comment:
There is no parsing required, so really confusing understanding the meaning of this method./

Reply:
Renamed the method to "ToString()", also made it public so that the user of the class can view the current verse without modifying it.